mappings/net/minecraft/client/gui/Selectable.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_6379 net/minecraft/client/gui/Selectable
 	METHOD method_37018 getType ()Lnet/minecraft/class_6379$class_6380;
-	METHOD method_37303 isSelectable ()Z
+	METHOD method_37303 isNarratable ()Z
```

Comment:
:laughing: I actually chose that name when I saw it in recipe book code while looking around